### PR TITLE
pythonPackages.pysonos: init at 0.0.11

### PR DESCRIPTION
**Pull request by oneingan**
pysonos package provides ability to control sonos media player using python. The original need was to use it with the home-assistant package, specifically the "sonos" component.

### CODE REVIEW

File: pkgs/development/python-modules/pysonos/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+, buildPythonPackage
+, fetchPypi
+, xmltodict
+, requests
+, ifaddr
+}:
+
+buildPythonPackage rec {
+  pname = "pysonos";
+  version = "0.0.11";
```
Comment (JohnAZoidberg) [inline pkgs/development/python-modules/pysonos/default.nix]: 0.0.12 is out, please upgrade.

**Comment by JohnAZoidberg** [comment]
Why did you remove the standard checkboxes from the PR message?

Please squash the commits. Hint: Use rebase instead of merge when you want to update your private branch.

**Comment by juaningan** [comment]
Please delete this. I mess it.

**Comment by juaningan** [comment]
new attempt in #60853

**Pull request closed without merging**